Dockerfile.core

```diff
@@ -0,0 +1,46 @@
+ARG OS_TAG=20.04
+FROM ubuntu:${OS_TAG} as builder
+
+ARG OS_TAG
+ARG BUILD_TYPE=release
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY ./ /root/bcc
+
+RUN apt-get -qq update && \
+    apt-get -y install build-essential libelf-dev libfl-dev clang libcap-dev
+
+RUN cd /root/bcc/src/cc/libbpf/src/ && make
+RUN cd /root/bcc/libbpf-tools/ && LLVM_STRIP=llvm-strip-10 make
+
+FROM ubuntu:${OS_TAG}
+
+RUN apt-get -qq update && \
+    apt-get -y install libelf1 libfl2
+
+# TODO: find a way to avoid specifying all the names
+COPY --from=builder                     \
+    /root/bcc/libbpf-tools/biolatency   \
```

Comment:
Doesn't this work?

```suggestion
COPY --from=builder                     \
    /root/bcc/libbpf-tools/*   \
```

Reply:
That would also copy all the source files. Actually I solved it by using `make install` to a specific folder on the builder image and then copying all the contents of that folder to the final image.